### PR TITLE
AST-492 - imrpv: Changed customizer submenu control title from "Container Animations" to "Submenu Animation"

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- https://wp-astra.atlassian.net/browse/AST-492
- Changed customizer submenu control title from "Container Animations" to "Submenu Animation".
- Changed for both Old and New Header footer builder.

### Screenshots
https://share.getcloudapp.com/JruD7dN4

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by imnavanath — DISMISSED** [review]

File: changelog.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ v3.5.1 (Unreleased)
 - Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
 - Improvement: Added close button for multiselect selected items in Google font variants.
+- Improvement: Changed customizer submenu control title from "Container Animations" to "Submenu Animation".
```
Comment (abhishek-bsf) [inline changelog.txt:6]: @sagarb90 Please update ``"Container Animations"`` to ``"Container Animation"``

Comment (sagarb90) [reply changelog.txt]: @abhishek-bsf Yes updated.

**Review by abhishek-bsf — DISMISSED** [review]

**Review by imnavanath — DISMISSED** [review]

File: changelog.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ v3.5.1 (Unreleased)
 - Fix: Cart Icon not working with Sticky Header after enabling the option "Hide when scrolling down" on mobile devices.
 - Fix: Secondary menu not expendable on mobile when it is outside of Off Canvas.
 - Improvement: Added close button for multiselect selected items in Google font variants.
+- Improvement: Changed customizer submenu control title from "Container Animation" to "Submenu Animation".
```
Comment (premanshup) [inline changelog.txt:6]:
```suggestion
- Improvement: Changed customizer submenu control title from 'Container Animation' to 'Submenu Animation'.
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: changelog.txt (hunk 2 above)
Comment (sagarb90) [reply changelog.txt]: Yes this is Done

**Review by premanshup — APPROVED** [review]

**Pull request merged**